recipes/struct-lmm/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "struct-lmm" %}
+{% set version = "0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 608cb82be883d640ef5490e513179984c0353d319c221af183b0dbe7b77af9f7
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >3
+    - setuptools >=39.0.1
+  run:
+    - chiscore >=0.2.0
+    - glimix-core >=3.1.7
+    - numpy >=1.15
+    - numpy-sugar >=1.5.0
+    - pytest >=4
```

Comment:
Is `pytest` needed as a runtime dependency?

Reply:
Yep, I have built-in tests that uses pytest framework: `struct_lmm.test()`.